pandas/tseries/index.py

```diff
@@ -308,6 +311,9 @@ def __new__(cls, data=None,
                                                      dayfirst=dayfirst,
                                                      yearfirst=yearfirst)
 
+        if data.dtype in (_INT64_DTYPE, _NS_DTYPE) and copy:
```

Comment:
hmm why is this copy here? (shouldn't it be in the ``tseries/index.py`` constructor?  (IOW this means it may copy multiple times, once here and once in the actual constructor)